src/commands/schema.ts

```diff
@@ -0,0 +1,60 @@
+import type yargs from "yargs";
+import type { Arguments, CommandBuilder } from "yargs";
+import fetch from "node-fetch";
+import { ChainName } from "@tableland/sdk";
+import getChains from "../chains";
+
+type Options = {
+  // Local
+  name: string;
+
+  chain: ChainName;
+};
+
+export const command = "schema <name>";
+export const desc = "Get info about a given table schema";
+
+export const builder: CommandBuilder = (yargs) =>
+  yargs.positional("name", {
+    type: "string",
+    description: "The target table name",
+  }) as yargs.Argv<Options>;
+
+export const handler = async (argv: Arguments<Options>): Promise<void> => {
+  const { name, chain } = argv;
+
+  // This isn't strictly required, because this REST API is chain agnostic.
```

Comment:
Note the comment here. This check isn't _really_ required, because this REST API is chain agnostic. However, right now in the SDK/CLI, we don't have another way to determine host url to use. Do we want to just hardcode the gateway in here?

Reply:
Interesting, would it make sense to let the caller pass in a value for host instead of chain?  If they don't we could just use the default network host (i.e. `getChains()["polygon-mumbai"].host`)?
This seems related to https://github.com/orgs/tablelandnetwork/discussions/240

Reply:
Yeh right, that's actually a pretty good call. Until we decide what to do here, perhaps we should just default to the default chain and ignore or go with the chain arg if it is there. Which... incidentally, is actually what this PR will already do 😉

Reply:
Boom!  consenting